contentdigest/digester_test.go

```diff
@@ -79,6 +79,12 @@ func TestDigester_HashRequest(t *testing.T) {
 			req:      io.NopCloser(mockReader{Err: errors.New("read error")}),
 			wantErr:  true,
 		},
+		{
+			name:     "empty",
+			digester: Empty,
+			req:      io.NopCloser(bytes.NewBufferString(`{"hello": "world"}`)),
+			want:     `empty=::`,
```

Comment:
I think what we actually want is just an empty string. I'll modify the `HashRequest` to bypass empty hasher